src/main/java/frc/robot/subsystems/Shoulder.java

```diff
@@ -94,7 +96,8 @@ public boolean onTarget() {
     }
 
     private void setSetpoint(double setpoint) {
-        setpoint = MathUtil.clamp(setpoint + this.shoulderOffset, ShoulderConstants.kMaxDistance, 360);
+        if (setpoint != 10)
```

Comment:
```suggestion
        if (setpoint != 10) {
```
Always use braces on ifs